docs/developer.md

```diff
@@ -1,4 +1,58 @@
 # Developer Landing Page
 
-If you develop or build software, here are some resources to help you
-better secure them.
+If you develop or build software, here are some ready-to-go
+resources to help you secure them from the
```

Comment:
this sounds like the software needs to be protected from the malicious actor `OpenSSF` 😅

Reply:
Ha! Good catch, that wasn't exactly the intended meaning :-). It's hard to word it briefly. I think I'll just move the reference to OpenSSF to make a "by the OpenSSF", as the author. That should be much clearer.